org.eclipse.jdt.launching/launching/org/eclipse/jdt/internal/launching/StandardVMType.java

```diff
@@ -808,7 +808,9 @@ public URL getDefaultJavadocLocation(File installLocation) {
 	 */
 	public static URL getDefaultJavadocLocation(String version) {
 		try {
-			if (version.startsWith(JavaCore.VERSION_24)) {
+			if (version.startsWith(JavaCore.VERSION_25)) {
```

Comment:
Java 24 block is missing now, also java 23 block below has link to Java 24, Java 23 to 22 etc. Should be cleaned up